core/block_dragger.js

```diff
@@ -429,7 +429,11 @@ const BlockDragger = class {
     // Moving icons moves their associated bubbles.
     for (let i = 0; i < this.dragIconData_.length; i++) {
       const data = this.dragIconData_[i];
-      data.icon.setIconLocation(Coordinate.sum(data.location, dxy));
+      // Don't move hidden Icons as they can have not been initialized.
+      // fixes #6076
+      if (!this.draggingBlock_.isCollapsed() || !data.icon.collapseHidden) {
```

Comment:
Do we still need this fix?

Reply:
Nope, core/block_dragger.js remains untouched. I thought i got rid of the changes I made in my first commit 🤔

Reply:
Wait! This code did trick my manual testing into false positive! More info in main thread... .